test/docker-e2e/e2e_upgrade_test.go

```diff
@@ -42,6 +42,10 @@ func (s *CelestiaTestSuite) TestCelestiaAppUpgrade() {
 			baseAppVersion:   4,
 			targetAppVersion: 5,
 		},
+		{
+			baseAppVersion:   5,
+			targetAppVersion: 6,
+		},
```

Comment:
test passes for me locally now, but v2-v3 can be flakey

maybe we can move some of these to the nightly since they shouldn't be changing